docs/0-intended-use/index.md

```diff
@@ -36,163 +36,39 @@ Original work by OpenRegulatory available at
-
-The {{device_name}} software is to be installed on a local server with one of
-the following OS types and versions:
-
-- Windows >= 10
-- Ubuntu LTS >= 22.04
-
-## Operating Principle
-
-<!-- > It's kind of a stretch to describe the "operating principle" of software. I
-> guess this makes more sense for hardware devices. In any case, I'd just
-> generally state what sort of input goes in and what output comes out, e.g.
-> you could be processing images and returning diagnoses. -->
-
-The {{device_name}} software is designed to fulfil the `Automated Contourer`
-role within the [IHE RO BRTO-II profile](http://ihe-ro.org/doku.php?id=doc:profiles:brto-ii).
-
-This is defined as:
-
-> A system that consumes [an image series] and creates an RT Structure Set. ...
-
-### To be utilised within the framework of standard DICOM input and output
-
-The software is intended to receive DICOM files that conform to the DICOM
-standard. The software that it provides results to is intended to receive and
-appropriately handle standards compliant DICOM structure files. It is not
-intended to directly interface with another medical device except via this
-standard DICOM input and/or output (either file based or DICOM network protocol
-based).
-
-### Prompt access to notifications, warnings and alerts
-
-Through the dashboard interface users can designate warning and alert
-thresholds for a range of relevant scenarios. The software itself may also
-present warnings and alerts. The email address of the user account used to
-access the software needs to be able to be readily accessed so that these
-configurable alerts and warnings can provide the appropriate notification.
+A Software as a Medical Device platform that supports patient treatment within
+the following use cases:
+
+- Clinical decision support within radiotherapy via automated contouring of
+  both target and avoidance structures within the treatment planning workflow.
+- Patient alignment and setup support during radiotherapy treatments via
+  patient surface guidance.
+- Treatment machine gating via patient surface tracking.
+- Alignment of radiotherapy simulation images into breathing phases to support
+  the treatment workflow.
+
+The {{device_name}} intends to be a platform that supports the safe and effective
+translation of open source research into Clinical Software as Medical Devices.
+As such, it is expected that future regulatory submissions will include
+expansions on the above intended use where the surrounding documentation is
+also updated accordingly.
+
+## Individual Components
+
+The {{device_name}} is split into separate components. Each component has its
+own documentation that covers the following items:
+
+- Intended medical indication
+- Contraindications
+- Patient population
+- User profile
+- Use Environment including software/hardware
+- Operating principle
+
+The documents for these individual components are the following:
+
+- [](autocontouring.md)
+- [](sgrt.md)
```

Comment:
The removed sections in this document have been moved into the new `autocontouring.md` document.